catalog/spatial/registry/registry-identification-plugin/src/main/java/org/codice/ddf/registry/identification/IdentificationPlugin.java

```diff
@@ -91,10 +124,70 @@ public CreateRequest process(CreateRequest input)
     @Override
     public UpdateRequest process(UpdateRequest input)
             throws PluginExecutionException, StopProcessingException {
+
+        OperationTransaction operationTransaction = (OperationTransaction) input.getProperties()
+                .get(Constants.OPERATION_TRANSACTION_KEY);
+
+        if (operationTransaction == null) {
+            throw new UnsupportedOperationException(
```

Comment:
Do you really want to throw an UnsupportedOperationException or would it be better to throw a PluginExecutionException (see https://github.com/codice/ddf/blob/9ff79f7312e1d0a2b5c600ffe197ad2a393e80df/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java#L1157 and https://github.com/codice/ddf/blob/9ff79f7312e1d0a2b5c600ffe197ad2a393e80df/catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/CatalogFrameworkImpl.java#L1101)